css/styles.css

```diff
@@ -0,0 +1,95 @@
+
+body {
+  background-color: #9370db;
+}
+
+h1 {
+  color: #c0c0c0;
+  font-size: 1.2rem;
+  margin-left: 20%;
+  margin-top: 15%;
+}
+
+h2 {
+  color: #fff;
+  font-size: 0.9rem;
+  padding-bottom: 20px;
+}
+
+p {
+  color: #fff;
+  font-size: 0.73rem;
+}
+
+.title {
+  height: 133px;
```

Comment:
For consistency, I'd suggest converting all your `px` sizes to `rem` so the relative sizes don't change.